cluster-autoscaler/cloudprovider/rancher/rancher_manager.go

```diff
@@ -207,8 +207,8 @@ func (m *RancherManager) DeleteNode(nodePoolId string, node *v1.Node) error {
 		return fmt.Errorf("node %s not found", node.Name)
 	}
 
-	klog.Infof("Marking node %s (%s) for deletion", rancherNode.ID, rancherNode.NodeName)
-	err = m.nodes.ActionPleaseKillMe(rancherNode)
+	klog.Infof("Scaling down node %s (%s)", rancherNode.ID, rancherNode.NodeName)
+	err = m.nodes.ActionScaledown(rancherNode)
```

Comment:
que tal nas chamadas para validar error colocar algo do tipo?
```suggestion
	if err = m.nodes.ActionScaledown(rancherNode); err != nil {
	    return err
	}
```
em alguns exemplos de go você vai achar nessa estrutura e acaba reduzindo linhas

Reply:
opa, acho que aí fica mais como uma escolha de projeto mesmo... no código do cluster-autoscaler em si a maior parte das verificações de erro - greppando aqui, eu diria que mais de 90% - deixam a declaração separada (eu mesmo acho que o código fica mais legível assim)